manifests/mongos/service.pp

```diff
@@ -25,6 +26,13 @@
   }
 
   if $service_manage {
+    if $facts['os']['family'] == 'RedHat' {
+      systemd::unit_file { 'mongos.service':
```

Comment:
why do we only manage it on redhat?

Reply:
First My work was only to fix the module for RedHat. Secondly I think other distro has the needed file in the package.